WordPress/src/main/java/org/wordpress/android/ui/comments/unified/UnifiedCommentListItemDecoration.kt

```diff
@@ -0,0 +1,90 @@
+package org.wordpress.android.ui.comments.unified
```

Comment:
This whole file was not part of the other PR. Is this file addition expected there?

Reply:
as mentioned in the PR description

>NOTE: code that was cherry picked from 17.7 was slightly ahead respect the original code in 17.6. This required adding the UnifiedCommentListItemDecoration that was added in 17.7. All this code is behind a feature flag, so as far as it compiles and we confirm the fix works we should be ok.

being the code behind a feature flag AFAIU if it compiles it should not have an impact since it is not run and we removed the dependency on the paging 3 lib (happy to have your thoughts on this as well @ashiagr 🙇 )

Reply:
Since it behind a feature flag + there are no compilation issues (and we need the new file after the latest changes since last fix) + no fast scrolling/ refresh issues, so 👍 from me.